src/securityPolicy.c

```diff
@@ -663,6 +663,31 @@ security_policy_t policyForSignTxValidityIntervalStart()
 	SHOW();
 }
 
+security_policy_t policyForSignTxMintDefault()
```

Comment:
I don't even know what this function should do. I'm completely in the dark about how the security policies should work in minting.

Reply:
In general, there are some security policies that do nothing at present (just allow to proceed).

For minting, the policy should take signing mode as an argument and decide if minting is allowed or not: since it is governed by a script, it should be allowed in multisig txs and forbidden in all the rest (ordinary, pool reg.).

Reply:
At present, that's all we want from the policy AFAIK.

Reply:
Where do I get the signing mode? Should that be the txUsecase in common_tx_data_t, just not implemented yet?